### ISSUE TITLE
Как документировать код?

### ISSUE THREAD

**Issue by EmillkaakillmE** [webpractik/bitrixoa#2]
Установка и генерация .yaml файла прошла успешно. Но не совсем понимаю, как писать документацию, кроме как верстать этот файл статично
Роуты все позадавал, как в инструкции, в .settings настройку выставил.. А дальше как?😅

Ожидал чего-то вроде https://justcodeit.ru/swagger-docs-dlya-api-na-laravel/
Создание документации с помощью комментариев прямо над методами/классами. А как в итоге не совсем понимаю
Скажите пожалуйста, такой вариант вообще возможен? Если да, то прошу, направьте хоть, куда копать)

**Comment by north-leshiy** [comment]
Да, все также, аннотируете методы контроллеров комментариями, генерируется документация.
